doc/articles/guides/xf-migration/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: Uno.XamarinFormsMigration.Overview
+---
+
+# Migrating from Xamarin.Forms to Uno Platform
+
+Apps that are built with native controls and APIs are able to meet today's standards for accessibility and performance. The launch of Xamarin.Forms in 2014 brought a UI framework to the .NET community that was intentionally designed to embrace this approach — across multiple platforms. Pages and layouts could be composed quickly using XAML, with developers rarely needing to write platform-specific code.
+
+Enter **Uno Platform**, a modern, cross-platform framework that shares Xamarin.Forms' commitment to leveraging native controls and APIs. However, it differs in key architectural areas which make up a compelling option to customize the presentation layer across app experiences'. This is especially valuable when your team needs to convey a precise brand identity. Developers evaluating Uno Platform may notice inherent similarities between the two, yet Xamarin.Forms has lagged behind other frameworks in its current state.
+
+Notwithstanding the [announced](https://dotnet.microsoft.com/platform/support/policy/xamarin#microsoft-support) end of support date in May 2024, Xamarin.Forms lacks the broad reach possible by targeting the web with the same codebase. Further, it's missed out on supporting features from major .NET releases that brought performance improvements and productivity gains.
+
+Fortunately, the commonalities with Xamarin.Forms make Uno Platform a natural choice for teams looking to modernize their existing investment. This straightforward series of articles will outline how to migrate key aspects of app to unlock the benefits of Uno Platform; meaning compatibility with a multitude of powerful controls, productivity gains from recent .NET features, and a demonstrated commitment to punctuality with support for work done upstream.
+
+With each article, we offer thoughtful insights that will aid adoption of Uno Platform as the path forward for existing your existing Xamarin.Forms investment.
+
+## What's covered
+
+* [Uno Platform for Xamarin.Forms developers](xref:Uno.XamarinFormsMigration.Intro)
+<!-- * [Controls](xref:Uno.XamarinFormsMigration.Controls)
+* [Animations](xref:Uno.XamarinFormsMigration.Animations) -->
```

Comment:
Does this need to stay commented?

Reply:
It does for now imo, because I'd prefer to do the content as a separate PR. 

However, I made changes to what's exposed in the overview:

https://github.com/unoplatform/uno/pull/13984/files#diff-8a958e4f972f0d2a18d2b6a2b66dde45f8aaad3ebdbcefdb6bf7f842604504dbR16-R35